### ISSUE TITLE
Print-function prints only None in (Python) Debug Console

### ISSUE THREAD

**Issue by AndyLuckert** [microsoft/vscode#88362]
**Problem:**
Yesterday, it still worked to have nicely-formatted printout of variables in the Debug Console during python debugging sessions, but now the built-in print-function doesn't do its job anymore apparently.

**Steps to Reproduce:**
Enter Debug-mode withcurrent python script via F5 and breakpoint-setting before in VS Code.
Then, switch to the Debug Console (via Ctrl+Shift+Y) and try to print out any of the currently stored variables and their associated values, just like so with my example variable "consum_uniques" in the following.

1. Ugly and unformatted print out:

```
consum_uniques
array(['2015-05-28T00:00:00.000000000', '2015-06-17T00:00:00.000000000',\n       '2015-07-07T00:00:00.000000000', '2015-07-21T00:00:00.000000000',\n       '2015-07-28T00:00:00.000000000', '2015-08-06T00:00:00.000000000',\n       '2015-08-17T00:00:00.000000000', '2015-09-01T00:00:00.000000000',\n       '2015-09-17T00:00:00.000000000', '2015-10-13T00:00:00.000000000',\n       '2015-10-27T00:00:00.000000000', '2015-11-10T00:00:00.000000000',\n       '2015-11-24T00:00:00.000000000', '2015-12-10T00:00:00.000000000'],\n      dtype='datetime64[ns]')
```

2. Supposedly nicely-formatted printout via the built-in print-function, which doesn't work anymore as it returns "None", but none of the variable content:
```
print(consum_uniques)
None
```

**- VSCode Version:**
1.41.1
26076a4de974ead31f97692a0d32f90d735645c0
x64

**- Python version:**
Python 3.7.6 (default, Jan  8 2020, 19:59:22) 
[GCC 7.3.0] :: Anaconda, Inc. on linux

**- OS Version:**
NAME="Ubuntu"
VERSION="18.04.3 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.3 LTS"
VERSION_ID="18.04"

**Comment by AndyLuckert** [comment]
Is there any way to find out quickly WHICH extension is responsible for that undesired behavior?
I have too many extensions installed to try them all out one by one.
Thanks in advance for your reply.

**Comment by weinand** [comment]
Python

**Comment by jjlkant** [comment]
For anyone else having encountering this issue: [issue 9540](https://github.com/microsoft/vscode-python/issues/9540) over on _vscode-python_ describes what happens.

**TLDR**: Python extension used to set the `redirectOutput` to `true` by default, which is no longer the case. In order to work around this, set `"redirectOutput": true` explicitly in launch.json

**Comment by AndyLuckert** [comment]
Thanks, with this workaround it works fine again.

**In my case, I edited the following part of the above-mentioned launch.json like so (basically by appending the line  "redirectOutput": true to the dict-entries:**
```
...
        {
            "name": "Python: Current File (Integrated Terminal)",
            "type": "python",
            "request": "launch",
            "program": "${file}",
            "console": "integratedTerminal",
            "redirectOutput": true
        },
...
```
Now, the print()-function works again while debugging in the Debug Console and even sends the printed output to both consoles at the same time.
